delfin/common/constants.py

```diff
@@ -104,6 +104,9 @@ class EventType(object):
     NOT_SPECIFIED = 'NotSpecified'
 
 
+# Default expiration time(in sec) for vmax connect request
+VMAX_EXPIRATION_TIME = 10
```

Comment:
Why is this 10? this is common timout for all requests ? some operation like volume fetching will take more time .

Reply:
this timeout is not for other operation like, volume or pool fetch. If we want timeout for these operations also(like vol/pool)..we need to add just another timeout variable.